packages/loopring_v3/test/testExchangeMaintenanceMode.ts

```diff
@@ -446,12 +442,12 @@ contract("Exchange", (accounts: string[]) => {
 
       let duration = 123;
       let expectedCost = await getDowntimeCost(duration);
-      let cost = await exchange.getDowntimeCostLRC(duration);
+      let cost = await getDowntimeCost(duration);
```

Comment:
The idea here was to compare the solidity implementation of the downtime cost calculation with a JS implementation. Now both just use the solidity code. But okay, we can fine-tune the tests later.

Reply:
will fix it later :)